hack/istio/install-service-spawner-demo.sh

```diff
@@ -50,7 +52,9 @@ install_service_spawner_demo() {
     ${CLIENT_EXE} create ns ${SSPAWNER}
   fi
 
-  ${CLIENT_EXE} label namespace ${SSPAWNER} istio-injection=enabled --overwrite=true
+  if [ "${AUTO_INJECTION}" != "" ]; then
```

Comment:
Shouldn't this be explicitly checking for true - first, to be consistent with all the other ones you added in the other scripts, and second, because this will inject sidecars even if someone sets the value to `"false"`:
```suggestion
  if [ "${AUTO_INJECTION}" == "true" ]; then
```